test_scripts/AppServices/GetAppServiceRecords/001_GetAppServiceRecords_MOBILE.lua

```diff
@@ -0,0 +1,73 @@
+---------------------------------------------------------------------------------------------------
+--  Precondition: 
+--  1) app1 is Registered
+--  2) AppServiceProvider permissions are assigned for <app1ID>
+--
+--  Steps:
+--  1) app1 sends a PublishAppService (with {serviceType="MEDIA", handledRPC=ButtonPress})
+--  2) HMI sends a GetAppServiceRecords to Core
+--
+--  Expected:
+--  1) Core returns the service record of the MEDIA service published by app1 ({servicePublished=true, serviceActive=true}) 
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/AppServices/commonAppServices')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local functions ]]
+local function PTUfunc(tbl)
+    local pt_entry = common.getAppServiceProducerConfig(1)
+    pt_entry.app_services.MEDIA = { handled_rpcs = {{function_id = 41}} }
+    tbl.policy_table.app_policies[common.getConfigAppParams(1).fullAppID] = pt_entry
+end
+
+local function getAppServiceRecords(serviceType, expectedResponse)
+    expectedResponse = expectedResponse()
```

Comment:
No need to pass expectedResponse as a parameter when you just call the local function anyways